zerver/models.py

```diff
@@ -299,13 +299,20 @@ def flush_realm_filter(sender, **kwargs):
 post_delete.connect(flush_realm_filter, sender=RealmFilter)
 
 class UserProfile(AbstractBaseUser, PermissionsMixin):
+    DEFAULT_BOT = 1
+    # that incoming webhook bots are limited to only sending messages via webhooks,
+    # and thus it is less of a security risk to expose their API keys
+    # since they can't be used to read messages
```

Comment:
Would write this as:

""
Incoming webhook bots are limited to only sending messages via webhooks.
Thus, it is less of a security risk to expose their API keys to third-party services, since they can't be used to read messages.
"""
￼